### ISSUE TITLE
DetailsList: Cells in Grouped DetailsList does not read out column header

### ISSUE THREAD

**Issue by kangka29** [microsoft/fluentui#13735]
### Environment Information

- **Package version(s)**: office-ui-fabric-react@v7.117.4
- **Browser and OS versions**: all

### Issue reproduced in codepen:
Export to codepen from [component page](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped) of Official Fluent UI website.

#### Actual behavior:
Screen reader not reading out column header when focussed on each cell.

#### Expected behavior:
Screen reader should read out column header when focussed on each cell.

#### Additional information:
The same issue has been referenced in bugs [here](https://github.com/microsoft/fluentui/issues/7560) & [here](https://github.com/microsoft/fluentui/issues/6019) but it has probably not been fixed for DetailsList - Grouped.

**Comment by paulgildea** [comment]
@xugao I referenced this issue here https://github.com/microsoft/fluentui/issues/7560, 

But let's use this one for tracking this issue for our a11y focus this coming iteration.

**Comment by smhigley** [comment]
This was resolved in #14835, should be fine to close now
